src/cmd/key/pass.sh

```diff
@@ -17,7 +17,11 @@ Try first:  $(basename $0) key join
       and:  $(basename $0) key split
 "
 
-    gpg --batch --no-tty --passwd $GPG_KEY
```

Comment:
This was returning 0 even if the password wasn't changed. Which was causing test to pass.